src/content/docs/ru/tutorial/6-islands/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+type: tutorial
+unitTitle: Поставьте курс на острова Astro
+title: 'Шаг 6 - Острова Astro'
+description: |-
+  Руководство: Создайте свой первый блог в Astro —
+  Используйте острова Astro для добавления компонентов фреймворка на ваш сайт в Astro
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Box from '~/components/tutorial/Box.astro';
+import Checklist from '~/components/Checklist.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import Option from '~/components/tutorial/Option.astro';
+import StackblitzIntro from '~/components/tutorial/StackblitzIntro.astro';
+
+
+Теперь, когда у вас есть полностью функционирующий блог, давайте добавим некоторые интерактивные острова на ваш сайт!
+
+## Где мы находимся сейчас?
+
+<StackblitzIntro tree="withastro/blog-tutorial-demo/tree/unit-6/start"/>
```

Comment:
Like the other chapters, you can translate the text directly here instead of using the component!